jars/sources/ExerelinCore/exerelin/campaign/intel/RespawnBaseIntel.java

```diff
@@ -305,7 +305,7 @@ public void makeKnown(TextPanelAPI text) {
 		if (getPlayerVisibleTimestamp() == null) {
 			Global.getSector().getIntelManager().removeIntel(this);
 			Global.getSector().getIntelManager().addIntel(this, text == null, text);
-		}
+		} // TODO: Not sure if I should queue this or not
```

Comment:
Shouldn't need queueing. The intel is only added here when the the player finds the base, in which case it _needs_ to be visible immediately for the notification to work.

Although why the pirate base intel (which is where I copied this from) removes the intel only to readd it immediately, I don't know.

Reply:
Ah, so that's what it's used for.